application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -99,7 +100,11 @@ analytics.livedata.action.rowEvolution.title=Row Evolution
 ## Others
 analytics.extension.name=Analytics Application (Pro)
 analytics.extension.title=Analytics
-</content>
+rendering.macro.analyticsFilter.startDate=Start date
+rendering.macro.analyticsFilter.endDate=End date
+rendering.macro.analyticsFilter.apply=Apply
+rendering.macro.analyticsFilter.name=Analytics Filter
+rendering.macro.analyticsFilter.description=Updates all the analytics macros on the same page as this macro to the period set by the user.</content>
```

Comment:
This needs some reformulations, it's not very clear
`Add filtering options for the analytics macros of the current page. The filtering criteria will be saved in the current URL, to have a reproducible state, and will also override eventual filter set at macro level.` - probably still needs improvements